### PR TITLE
Hardware canary one

### PR DESCRIPTION
**Pull request by hdngr**
Adding canary-one to devices page.

### CODE REVIEW

File: docs/hardware/devices/index.mdx [hunk 1]
```diff
@@ -25,6 +25,13 @@ Please do your research and choose the board that meets your needs (or maybe alr
 - ESP32-based devices require more power to operate but are typically lower-cost alternatives that do perform well when using house power, or for handsets that only require a day or two of runtime, and for applications that require WiFi connectivity.
 :::
 
+### [CanaryOne](./canary/)
+Complete solution with battery, screen, case, and antenna.  Ships pre-flashed with latest Meshtastic firmware.
+
+| Name                                           | MCU      | Radio  |     WiFi     | BT  |  GPS   |
+|:-----------------------------------------------|:---------|:-------|:------------:|:---:|:------:|
+| [CanaryOne](./canary/canary-one)               | nRF52840 | SX1262 |      NO      | 5.0 | YES    |
```
Comment (pdxlocations) [inline docs/hardware/devices/index.mdx:33]: I think is is a broken link

**Review by pdxlocations — CHANGES_REQUESTED** [review]
There's a broken link causing the build to fail

File: docs/hardware/devices/index.mdx (hunk 1 above)
Comment (hdngr) [reply docs/hardware/devices/index.mdx]: @pdxlocations thanks for the quick review.  fixed.

**Comment by pdxlocations** [comment]
@tekstrand or @rcarteraz could you  hit this one up with a Vercel preview?  Lol, I really should bug @sachaw to add me to the vercel team.

File: docs/hardware/devices/canary/index.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Specifications
+
+- **MCU**
+  - NRF52840 (Bluetooth)
+    - Bluetooth 5.0
+- **LoRa Transceiver**
+  - Semtech SX1262
+- **Frequency options**
+  - All Mesthastic options.
```
Comment (pdxlocations) [inline docs/hardware/devices/canary/index.mdx:25]: Typo "Mesthastic"

**Review by pdxlocations — CHANGES_REQUESTED** [review]
Just one last fix and this should be golden.  Very nice!

**Comment by sachaw** [comment]
@pdxlocations please DM me your email, and I'll send an invite

File: docs/hardware/devices/canary/index.mdx (hunk 2 above)
Comment (hdngr) [reply docs/hardware/devices/canary/index.mdx]: In my find all, found another spot where mesthastic was mispelled, so fixed that too.  We could consider implementing cspell.  I might add that in another PR.

Comment (pdxlocations) [reply docs/hardware/devices/canary/index.mdx]: it has proven to be a very difficult word to spell properly.

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**